backend/src/main/java/ba/unsa/etf/si/bbqms/ws/controllers/TellerStationController.java

```diff
@@ -153,6 +154,22 @@ public ResponseEntity getBranchStations(@PathVariable final String tenantCode,
         }
     }
 
+    @GetMapping("/{stationId}/tickets")
+    public ResponseEntity getTicketsForTellerStation(@PathVariable final long stationId) {
+        try {
+            final Set<Service> services = stationService.getServicesForTellerStation(stationId);
+            final long branchId = stationService.getBranchIdForTellerStation(stationId);
```

Comment:
Why are we fetching the `branchId`? TellerStation has `Branch` attribute which we can use.